src/webhooks/registry.ts

```diff
@@ -104,6 +98,12 @@ function buildCheckQuery(topic: string): string {
             ... on WebhookEventBridgeEndpoint {
               arn
             }
+            ${versionSupportsPubSub()
```

Comment:
Yes it's a bit of a mess... unfortunately the GraphQL endpoint returns an error if we don't have this check. For older API versions which don't support EventBridge, it uses `legacyQuery` instead, which is why it doesn't error out, and why we don't need the check there. I think this could use a refactor (we should build a completely separate query for each delivery method?) but didn't want to risk touching too much in this PR.